default.nix

```diff
@@ -68,6 +70,13 @@ pkgs.lib.fix (self: rec {
         chmod +w ./yarn.lock
 
         yarn config --offline set yarn-offline-mirror ${offlineCache}
+        yarn config --offline set yarn-offline-mirror-pruning false
+        yarn config set yarn-offline-mirror-pruning false
```

Comment:
I noticed that yarn was attempting to unlink some packages in the offline mirror and I thought it was because it was trying to prune unused packages. However, this config setting didn't resolve the problem, so maybe there's another, yet-to-be understood reason.

Reply:
Maybe it's the yarn 1.2.0 update. Do you know what version was used?

Reply:
I was using yarn `1.2.1`. Do you see anything fishy in https://github.com/yarnpkg/yarn/releases ?